src/wabt.post.js

```diff
@@ -86,28 +100,24 @@ function allocateCString(s) {
 /// Features
 function Features(obj) {
   this.addr = Module._wabt_new_features();
-  for (var i = 0; i < FEATURES.length; ++i) {
-    var feature = FEATURES[i];
-    this[feature] = obj[feature] | 0;
-  }
+  allFeatures().forEach((v, f) => this[f] = booleanOrDefault(obj[f], v));
```

Comment:
How about `for (const [f, v] of Object.entries{FEATURES}) { ... }`   ?

Reply:
Emscripten just [doesn't like it](https://github.com/WebAssembly/wabt/actions/runs/3784684407/jobs/6434167954#step:3:1025), I don't know why.

Reply:
think I found a way around it, dropping the `const` :shrug: